README.md

```diff
@@ -3,16 +3,17 @@
 # For dataset creators:
 ## To create a new dataset:
 #### Option 1: Use the `cell_feature_data` package
-1. Make a branch or fork of this repo
-2. Create a virtual environment if not already created: `python3 -m venv venv/[ENV-NAME]`
-3. Activate the virtual environment: `source [ENV-PATH]/bin/activate`
-4. Install the dependencies: `pip install -e .` This step also makes the `create-dataset` command available globally within the virtual environment.
-5. Run `create-dataset` to start the dataset creation process. This will: 
+1. Create a virtual environment if not already created: `python3 -m venv venv/[ENV-NAME]`
```

Comment:
```suggestion
1. Create a virtual environment if not already created: `python3 -m venv [ENV-PATH]`
```
Suggesting this to match the following step 2.  What do you think?

Reply:
ah good catch! Fixed!